cmd/syncv3/main.go

```diff
@@ -113,10 +120,53 @@ func main() {
 		AddPrometheusMetrics: args[EnvPrometheus] != "",
 	})
 
+	if args[EnvSentryDsn] != "" {
+		fmt.Printf("Configuring Sentry reporter...\n")
+		err := sentry.Init(sentry.ClientOptions{
```

Comment:
Why isn't this with the other `if args[EnvSentryDsn] != "" {` code? I'm guessing we need to have init'd it prior to calling `go h2.StartV2Pollers()` - but why? Please comment.

Reply:
Two concerns:

- in an upcoming PR, I'm going to make any error messages from `h2.StartV2Pollers` also log to sentry, so I want to init before that.
- I think I was worried  that the order of installing the otel and Sentry handlers might be important. On reflection I'd hope that the two middlewares are orthogonal.

More generally: Sentry has some support for tracking and tracing; I wonder if there's some kind of overlap or duplicated effort here.

Reply:
Commented in https://github.com/matrix-org/sliding-sync/pull/57/commits/646232dcb0e6f6daa82dc8dd7770cbf121bac001.